accord-core/src/main/java/accord/impl/InMemoryCommand.java

```diff
@@ -41,7 +41,7 @@ public class InMemoryCommand extends Command
     private PartialTxn partialTxn;
     private Ballot promised = Ballot.ZERO, accepted = Ballot.ZERO;
     private Timestamp executeAt;
-    private PartialDeps partialDeps = PartialDeps.NONE;
+    private @Nullable PartialDeps partialDeps = null;
```

Comment:
So PartialDeps.NONE could be confused with something else? Like empty dependencies?